requirements/extras/lzma.txt

```diff
@@ -1,2 +1,2 @@
---extra-index-url https://pypi.anaconda.org/nehaljwani/simple # Temporary index for Windows wheels
+# --extra-index-url https://pypi.anaconda.org/nehaljwani/simple # Temporary index for Windows wheels
```

Comment:
If this is not needed anymore, maybe we should just remove the line?

Reply:
I don't know that it isn't needed; I don't have access to a Windows machine. All I know is that it doesn't seem to work and it breaks the build on my Linux machine.

Reply:
@shaib if Travis CI passes without this change, then I think it should be the subject of a different PR. What do you think?

Reply:
I'm not sure this change is necessary at all. When I tried at the office earlier, I consistently got a 404 for the index page, so I figured the URL was no longer valid. I just tried from home and it seems to be responding with content, so... this change is a separate commit, which should probably be dropped.

Reply:
The index loads for me now -- it appears it is a set of wheels packages for backports.lzma whereas no wheels files exist in pypi for backports.lzma.   We will want to maintain this line -- perhaps your office DNS was not responding to the index or it was temporarily down.   We require backports.lzma for python < 3.3

Reply:
Change removed.